compiler/src/linking/link.re

```diff
@@ -339,7 +307,9 @@ let link_all = (linked_mod, dependencies, signature) => {
   table_offset := 0;
 
   let link_one = dep => {
-    let local_names: Hashtbl.t(string, string) = Hashtbl.create(128);
+    let local_functions: Hashtbl.t(string, string) = Hashtbl.create(128);
+    let local_globals: Hashtbl.t(string, string) = Hashtbl.create(128);
```

Comment:
```suggestion
    let module_globals: Hashtbl.t(string, string) = Hashtbl.create(128);
```
or something similar? `local_globals` really throws me through a loop (same thing for `comp_utils.re`)

Reply:
It's definitely a funny-sounding name 🙂 

Will update it.